substrate/primitives/core/src/lib.rs

```diff
@@ -47,10 +47,12 @@ pub use sp_debug_derive::RuntimeDebug;
 pub use impl_serde::serialize as bytes;
 
 #[cfg(feature = "full_crypto")]
-pub mod hashing;
+#[deprecated(
+	since = "27.0.0",
+	note = "`sp-crypto-hashing` re-exports will be removed after June 2024. Use `sp-crypto-hashing` instead."
```

Comment:
```suggestion
	note = "`sp-crypto-hashing` re-exports will be removed after June 2024. Use `sp-crypto-hashing` directly instead."
```

Reply:
nvm - I didn't see that this is from a canonical template.
https://github.com/paritytech/polkadot-sdk/blob/caa987d26d14bbdbfbc9be08a1da4c2b117fcaa3/docs/contributor/DEPRECATION_CHECKLIST.md#hard-deprecate-by-adding-a-warning-message